examples/Dojo/index.html

```diff
@@ -87,49 +89,16 @@ <h2>Checkout Input Form</h2>
     <script>
       dojoConfig = {
         baseUrl: '.',
-        paths: {
-          js: "js"
-        },
-        packages: [
-          {
-            name: 'rci/sdk',
-            location: 'rci/sdk.js',
-            lib: '.'
-          },
-          {
-            name: 'rci/producerFactory',
-            location: 'rci/producerFactory.js',
-            lib: '.'
-          },
-          {
-            name: 'rci/stateService',
-            location: 'rci/stateService.js',
-            lib: '.'
-          },
-          {
-            name: 'rci/actionService',
-            location: 'rci/actionService.js',
-            lib: '.'
-          },
-          {
-            name: 'rci/onLoadService',
-            location: 'rci/onLoadService.js',
-            lib: '.'
-          }
-        ],
-        paths: {
-          jquery: "lib/jquery-3.4.1.min.js"
-        }
+        async: true
       }
     </script>
 
     <!-- Dojo app -->
-    <script src="js/lib/jquery-3.4.1.min.js""></script>
     <script src="js/rci/sdk.js"></script>
     <script src="js/rci/producerFactory.js"></script>
     <script src="js/rci/actionService.js"></script>
     <script src="js/rci/stateService.js"></script>
-    <script src="js/rci/onLoadService.js"></script>
+    <script src="js/rci/readyService.js"></script>
```

Comment:
Because we now have async: true on the dojoConfig, these dependencies shouldn't be required here as the dojo loader will pull them in on demand as it encounters the references in app.js

Reply:
I've left them in.  Tried an alternative configuration by reading the docs you provided but couldn't get it to hang together.